flow/project.py

```diff
@@ -330,13 +468,18 @@ def __init__(self, id, name, job, cmd, directives=None):
         self.directives._keys_set_by_user = keys_set_by_user
 
     def __str__(self):
-        return "{}({})".format(self.name, self.job)
+        if len(self.jobs) == 1:
+            return "{}({})".format(self.name, str(self.jobs[0]))
+        else:
+            return "{}({}...{}...{})".format(
+                self.name, str(self.jobs[0])[0:8], len(self.jobs), str(self.jobs[-1])[0:8]
```

Comment:
The format is not bad, but I would personally rather `name-job-number-ids`. We should see others opinions first though.

Reply:
What is the purpose of this, and would it make more sense just to have `name-number` since the job ids are not actually very helpful in this case?

Reply:
I meant `name-number-ids`, but Alyssa's suggestion works too, though it will be impossible to distinguish between aggregates with the same number. We are already working with that difficulty in general since the name with the ids would get much too long quickly.